polyphemus/Gemfile.lock

```diff
@@ -1,7 +1,5 @@
-GIT
-  remote: https://github.com/mountetna/monoetna.git
-  revision: 6f7c04889f55ef2d38e147a4abbb2d0d2a44dfc2
-  branch: refs/artifacts/gem-etna/c8091f573c914e6db2f4c3d09eed812782a5670e
+PATH
```

Comment:
This will get fixed in the github action that runs after merge to master.  
TODO:  This magic will go away once I complete the docker deployment work, since we can always just us the path reference to the gem for applications.